parquet-column/src/main/java/org/apache/parquet/column/impl/ColumnWriterBase.java

```diff
@@ -409,4 +428,14 @@ abstract void writePage(
       ValuesWriter definitionLevels,
       ValuesWriter values)
       throws IOException;
+
+  abstract void writePage(
+      int rowCount,
+      int valueCount,
+      Statistics<?> statistics,
+      SizeStatistics sizeStatistics,
```

Comment:
There could be some confusion of the two names that sieStatistics is one type of statistics but we are separating them

Reply:
I agree. As they are also defined in the thrift file, I think it is not a big problem.